k8s-dev/cstor-pool-mgmt/pool-resource-cluster-1.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: openebs.io/v1alpha1
+kind: CStorPool
+metadata:
+  name: pool1-abc
+  node: node-host-label
```

Comment:
can we remove the node from metadata?
let metadata be the usual k8s metadata struct

Reply:
will change